tree/dataframe/inc/ROOT/RDFNodes.hxx

```diff
@@ -278,7 +278,7 @@ class TColumnValue {
 public:
    static constexpr bool fgMustUseRVec = MustUseRVec;
 
-   TColumnValue() = default;
+   TColumnValue() {};
```

Comment:
Why is this change needed?

Reply:
If I don't do this, I get missing symbols on ubu 16 gcc5 :(